main.go

```diff
@@ -64,7 +65,10 @@ func loggingMiddleware(next http.Handler) http.Handler {
 
 func generateNames(count int, seed string) ([]string, error) {
 	var allNames []string
-	namer := NewNameifier()
+	namer, err := lib.NewNameifier()
+	if err != nil {
+		return nil, err
```

Comment:
Do we need to wrap some context on this error? Or is the passed message clear enough?